posthog/api/test/test_personal_api_keys.py

```diff
@@ -160,13 +160,21 @@ def test_header_resilient(self):
         )
         self.assertEqual(response.status_code, 200)
 
+        # Retrieve key from db to check that no update was made
+        key = PersonalAPIKey.objects.get(id=self.key.id)
+        self.assertEqual(key.secure_value, self.key.secure_value)
```

Comment:
Personally I would prefer that we pull `self.key.secure_value` out before we do the API call. I know that the object doesn't get refreshed but given how important the test is it would be cool to be explicit about it